src/main/java/org/apache/log4j/Logger.java

```diff
@@ -54,15 +54,15 @@
   */
   static
   public
-  Logger getLogger(String name) {
+  Logger getLoggerID(String name) {
```

Comment:
Whats wrong/irritating about `#getLogger(String)`?
IMHO `#getLoggerID(String)` is much more misleading, since the return type of this method is a `Logger`, not an ID ...